sycl/include/CL/sycl/detail/scheduler/commands.cpp

```diff
@@ -92,7 +92,7 @@ void ExecuteKernelCommand<
   }
 
   if (m_KernelArgs != nullptr) {
-    unsigned ArgumentID = 0;
+    uint32_t ArgumentID = 0;
```

Comment:
This change seems to be unrelated to the fix.
Suggest reverting.

Reply:
This change IS related to the fix.
The fix changed the 1st argument of passGlobalAccessorAsArg() and passLocalAccessorAsArg from uint to uint32_t and ArgumentID is passed to there.

BTW, your comment helped me to find couple places where I did not move the patch properly from WIndows (where it was complete) to Linux (where I also tested the patch and from where uploaded fix for review). Those 2 places are fixed now (changed uint to uint32_t there).